master/docs/Makefile

```diff
@@ -6,6 +6,9 @@ dev: clean html
 
 VERSION := $(shell if [ -n "$$VERSION" ]; then echo $$VERSION; else PYTHONPATH=..:$${PYTHONPATH} python -c 'from buildbot import version; print(version)'; fi)
 
+# sphinx is fastest on Buildbot docs when using around 8 cores. Anything more does not improve the build speed
```

Comment:
then we want to use max, no?

Reply:
It is exactly this, min/max are not intuitive in this context, function like clamp is better.

Say, someone has 12 cores, min(8, 12) == 8. If someone has 1 core, min(8, 1) == 1.

Reply:
obviously :}